src/reconstruction/modifications/generic.jl

```diff
@@ -2,9 +2,9 @@
     with_set_bound(args...; kwargs...) 
 
 Specifies a model variant that has a new bound set. Forwards arguments to
-[`set_bound`](@ref). Intended for usage with [`screen`](@ref).
+[`change_bound`](@ref). Intended for usage with [`screen`](@ref).
```

Comment:
plural variant that would call `change_bounds` is missing?

Reply:
adding it